crates/red_knot_workspace/resources/test/empty-test-venv/bin/activate

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2020-202x The virtualenv developers
```

Comment:
Personally I'd be inclined to remove some totally-irrelevant files from the virtualenv (all the activate scripts), but it's not a big deal either way.